templates/glanceapi

```diff
@@ -0,0 +1 @@
+glance
```

Comment:
not sure where to comment on the actual service config. therefore I add it here. The `glance-api.conf` is missing a condition for the actual need of the two api instances. default to `show_image_direct_url=False` and only have the internal one to `show_image_direct_url=True`.

Reply:
Okay, I will add that to the base conf file and then use `APIType` from the spec to set the proper value in the template parameter.

Reply:
Done